### DISCUSSION TITLE
Click that makes game window active again shouldn't close overlays

### DISCUSSION THREAD

**Discussion by LittleEndu** [ppy/osu#13121] [Ideas]
**Screenshots or videos showing encountered issue:** 
https://youtu.be/dtAxbmBEHJ0

**osu!lazer version:** 
2020.408.1

**Comment by Joehuu** [comment]
This will be fixed once we move to SDL (or append `--sdl` atm).

**Reply by peppy** [reply]
@Joehuu confirm that this is resolved?

**Reply by peppy** [reply]
This is indeed resolved.

**Comment by peppy** [comment]
not sure i agree with this

**Comment by Joehuu** [comment]
Fixed in SDL as of now, although I agree this is non-standard behavior so leaving open for now.

**Comment by LittleEndu** [comment]
I assume this should be converted back to an issue if it's agreed upon that the current behaviour is unwanted

**Comment by peppy** [comment]
This has since been resolved via the SDL2 switch.
